### PR TITLE
TLS client verification

### PR DESCRIPTION
**Pull request by silverdaz**
Client verification for TLS connections

### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [x] New feature
- [x] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Related issues:
#3 

### Additional information:
This includes connection to (fake) CentralEGA users

### CODE REVIEW

**Review by jbygdell — APPROVED** [review]

File: src/mq-config.h [hunk 1]
```diff
@@ -22,7 +22,9 @@ struct mq_options_s {
   int   ssl;
   int   verify_hostname;
   int   verify_peer;              /* For the SSL context */
-  char* cacert;                   /* For TLS verification */
+  char* cacertfile;               /* For TLS server verification */
```
Comment (jbygdell) [inline src/mq-config.h:25]: Changing `cacert` to `cacertfile` requires a similar change in sshd.c, line 1702 to be precise.

**Review by jbygdell — CHANGES_REQUESTED** [review]

File: src/mq-config.h (hunk 1 above)
Comment (silverdaz) [reply src/mq-config.h:25]: I thought I had updated that already.
I checked again.
Is that comment just old?

Comment (jbygdell) [reply src/mq-config.h:25]: Look in `src/openssh/sshd.c` line 1702 , when I tried to build that branch this morning it failed because of that.

Comment (jbygdell) [reply src/mq-config.h:25]: https://github.com/EGA-archive/LocalEGA-inbox/blob/3e56e889725fc0943254a39cadb0c8ce91fddc55/src/openssh/sshd.c#L1702

Comment (silverdaz) [reply src/mq-config.h:25]: My bad. I had forgotten to push the latest version.

**Review by jbygdell — APPROVED** [review]

**Review by omllobet — APPROVED** [review]

**Pull request merged**